rebar.config

```diff
@@ -1,8 +1,6 @@
 {require_otp_vsn, "R1[456]|^[0-9]+"}.
 
-{deps, [
-    {jsonx, ".*", {git, "https://github.com/iskra/jsonx.git", {branch, master}}}
-]}.
+{deps, [{jsonx, "", {git, "https://github.com/alertlogic/jsonx.git", {branch, master}}}]}.
```

Comment:
rebar3 doesn't need the version regex in the location tuple.

```erlang
{jsonx, {git, "https://github.com/alertlogic/jsonx.git", {branch, master}}}
```
I think dropping encoding and so forth is a good idea.